src/components/FormBuilder.tsx

```diff
@@ -280,7 +287,7 @@ export class FormBuilder extends React.Component<IFormBuilderProps, {}> {
 
         const component = React.createElement(fieldDef.builder, fieldBuilderProps);
         return (
-            <div className='form-builder-field' key={field.id}>
+            <div ref={this.attachFieldRef} className='form-builder-field' key={field.id}>
```

Comment:
One more question, if we are loading an existing template, all fields will trigger AttacheFieldRef. Then the last one will be scrolled into view? Will this behavior be a little strange?